Colours.txt

```diff
@@ -6,19 +6,20 @@
 #define centrelinecolour 15790135
 #define coast 9076039
 ; @preserveComment Alternative colours for the above line, change to your preference. STC/PC (Grey) 5324604 Yellow 32896
-#define col_gate 3953980
 #define Edge 6589540
-#define Filled 6316128
 #define fire 2894694
+#define geoDefault 5787205
 #define grass 6919785
 #define grey 10534048
+#define JJCAS 7484470
+#define JJLand 6189439
+#define JJOCAS 6249039
 #define land 3947580
 ; @preserveComment Alternative colours for the above line, change to your preference. Light 8158332 Dark 1777181
 #define LTMACAS 4915200
 #define motorway 2631720
-#define NXsidlines 255
 #define NXlanes 12632256
-#define OCASregion 0
```

Comment:
For those that want to customise the OCASregion, I think this should be retained. Or change OCASregionGrey to OCASregion and only have one 'Uncontrolled Airspace' region with the grey colour.

Reply:
Sorry not quite sure what you are suggesting. Keep OCASregion defined as black and add a new "OCASgrey" definition?
And how does that reduce the number of lines in the sf?

Reply:
@RedstonePilot There is already an `OCASregionGrey` defined.

I hadn't realised people might want to customise the OCAS region colour, so if that is the case, I think simply keep the OCASregion defition as Sam suggests.

The line count I think Sam means we could in theory have only one file for OCAS Airspace (we currently have two - one black and one grey). But I wouldn't prioritise revmoving a ~450 line file over functionality personally.